src/sequence-steps.js

```diff
@@ -54,19 +97,44 @@ export function gitBranchGrepUpstreamDevelop( [ git, options ] ) {
 	} );
 }
 
+export function askPrereleaseIdentifier( [ git, options ] ) {
+	if ( options.tag && options.tag.length ) {
+		return Promise.resolve();
+	}
+
+	return utils.prompt( [ {
+		type: "input",
+		name: "prereleaseIdentifier",
+		message: "Prerelease Identifier:"
+	} ] ).then( answer => {
+		options.tag = answer.prereleaseIdentifier;
+		return Promise.resolve();
+	} );
+}
+
 export function gitCheckoutMaster( [ git, options ] ) {
 	const command = "git checkout master";
 	utils.log.begin( command );
 	return nodefn.lift( ::git.checkout )( "master" )
 		.then( () => utils.log.end() );
 }
 
-export function gitMergeUpstreamMaster( [ git, options ] ) {
-	const command = "git merge --ff-only upstream/master";
+export function getFeatureBranch( [ git, options ] ) {
+	const command = "git rev-parse --abbrev-ref HEAD";
+	utils.log.begin( command );
+	return utils.exec( command ).then( branch => {
+		options.branch = branch.trim();
+		utils.log.end();
+	} ).catch( () => utils.advise( "getFeatureBranch" ) );
```

Comment:
`advise` may or may not be needed at this point. for onesies the key doesn't exists, for twosies i'm not sure why this would fail other than git not being installed and i think git has been used before this point.